src/pages/CreateRoute/RouteManagement/components/Map/RoutingComponent.tsx

```diff
@@ -32,23 +39,39 @@ const RoutingComponent: React.FC<RoutingComponentProps> = ({ locations }) => {
     }
   };
 
-  const handleCalculateRoute = async () => {
+  const handleCalculateRoute = async (): Promise<void> => {
     try {
       const coordinates = await Promise.all(
-        locations.map((location) => geocode(location))
+        ['New York', ...locations].map((location) => geocode(location))
       );
 
-      L.Routing.control({
+      const routingControl = L.Routing.control({
         waypoints: coordinates.map((coords) =>
           L.latLng(coords.lat, coords.lon)
         ),
         routeWhileDragging: true,
         createMarker: () => null,
         lineOptions: {
-          styles: [{ color: `${COLORS.green}`, weight: 4 }],
+          styles: [
+            {
+              color: `#${Math.floor(Math.random() * 16777215)
```

Comment:
move it to function in utils and call - generateRandomColor

Reply:
done